supplementary/devices/nicki.rst

```diff
@@ -0,0 +1,77 @@
+
+Nicki - Sony Xperia M
+===========================
+
+Xperia M is very weak, especially by today's standards. But it runs Halium. Somehow.
+
+Status
+------
+
+Halium
+^^^^^^
+
+There is an ongoing port of Halium for Nicki that `Konrad Dybcio <https://github.com/ewentual>`_ is working on. Halium-7.1 branch builds with slightly tweaked LOS 14 sources (manifest will be published soon). Halium rootfs can be flashed using `halium-install by JBB <https://github.com/JBBgameich/halium-install>`_, yet /data partition is too small (2 gigs) to fit a plasma mobile rootfs.
+
+Distributions
+^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Distribution
+     - Device Specific Files
+     - Kernel
+     - What works
+     - What doesn't work
+   * - `LineageOS <https://wiki.lineageos.org/devices/nicki>`_
+     - `android_device_sony_nicki <https://github.com/lineageos/android_device_sony_nicki>`_
+     - `android_kernel_sony_msm8x27 <https://github.com/lineageos/android_kernel_sony_msm8x27>`_ based on v3.4.0
+     - Everything(afaik)
+     - Nothing
+   * - Halium Reference Rootfs
+     - `ewentual/halium_device_sony_nicki <https://github.com/ewentual/halium_device_sony_nicki>`_
+     - `ewentual/halium_kernel_sony_msm8x27 <https://github.com/ewentual/halium_kernel_sony_msm8x27>`_ based on v3.4.0
+     - Boots to rootfs, ssh works, test_vibrator works
```

Comment:
You probably won't ever open that document again. I'd prefer if you replaced it with "see projectmanagement issue"